web/src/HTTPconfig.js

```diff
@@ -8,8 +8,8 @@ REACT_APP_API_POINT_PORT=$ADMIN_EXT_PORT
 for local development, no need to source the env.sh or .env.
 */
 
-const adminHost = process.env.REACT_APP_API_POINT_HOST
-const adminPort = process.env.REACT_APP_API_POINT_PORT
+const adminHost = 'http://ncrs.d2.comp.nus.edu.sg'
+const adminPort = '3000'
 
```

Comment:
@naili-xing Could you merge this PR if it is okay to change 
process.env.REACT_APP_API_POINT_HOST to 'http://ncrs.d2.comp.nus.edu.sg'

Reply:
@arielclj I think maybe you can comment out the old line "const adminHost = process.env.REACT_APP_API_POINT_HOST"
and "const adminPort = process.env.REACT_APP_API_POINT_PORT" to reserve for other use in the future

Reply:
so it read like
// const adminHost = process.env.REACT_APP_API_POINT_HOST
// const adminPort = process.env.REACT_APP_API_POINT_PORT
const adminHost = 'http://ncrs.d2.comp.nus.edu.sg'
const adminPort = '3000'

Reply:
this is because it may lead to problem if the api point is not on ncrs port 3000, it could be elsewhere

Reply:
but the change is convenience to developer who do the testing with UI only (e.g. at panda.d2.comp.nus.edu.sg), while using the API point at ncrs:3000

Reply:
@chrishkchris included previous adminHost and adminPort in latest commit

Reply:
ok thanks